IR-1366: Update /docs/api/basics/#api-limits to include the new default /users/track rate-limit (#1184)
Appboy/braze-docs: _docs/_api/basics.md

|Default API Rate Limit | Value|
|---|---|

> |Requests to the `/users/track` endpoint|50,000 API requests per minute, though this is subject to change as noted below. |

Author: https://www.braze.com/docs/api/basics/#api-limits currently shows /users/track as Unlimited, this will document the default for new customers

https://www.braze.com/docs/api/endpoints/user_data/post_user_track/#user-track-responses does not currently mention rate-limits other than the possible error code, @dulle-braze should it mention the new default limit?

> |Requests to the `/users/track` endpoint|Default of 50,000 API requests per minute for Dashboard companies created after June 2, 2020 and default of Unlimited API requests for Dashboard companies created before then. Though this is subject to change as noted below. |

Reviewer: ```suggestion
|Requests to the `/users/track` endpoint|Default of 50,000 API requests per minute for Dashboard companies created after June 2, 2020 and default of Unlimited API requests for Dashboard companies created before then. These limits are subject to change as noted below. |
```
